assertk-jvm/src/test/kotlin/test/assertk/JVMAssertBlockTest.kt

```diff
@@ -0,0 +1,45 @@
+package test.assertk
+
+import assertk.assert
+import assertk.assertions.isGreaterThan
+import assertk.assertions.isNotNull
+import assertk.assertions.isPositive
+import org.junit.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFails
+
+class JVMAssertBlockTest {
+
+    private val errorSubject = assert { throw Exception("test") }
+
+    //region thrown error
+    @Test fun returnedValue_exception_fails_with_stacktrace() {
+        val error = assertFails {
+            errorSubject.returnedValue {
+                isPositive()
+            }
+        }
+
+        assert(error.message).isNotNull {
```

Comment:
I know it's not as nice, but been trying to keep assertk out of the assertions that test it.